src/views/explore/explore.jsx

```diff
@@ -0,0 +1,159 @@
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var React = require('react');
+var render = require('../../lib/render.jsx');
+
+var Api = require('../../mixins/api.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+var Tabs = require('../../components/tabs/tabs.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var Grid = require('../../components/grid/grid.jsx');
+
+require('./explore.scss');
+
+// @todo migrate to React-Router once available
```

Comment:
Yes I tried out using `require.ensure` for dynamic script loading yesterday and it seemed to work pretty well.  Although I believe we wouldn't even need to use that in this case since all of the explore pages use the same javascript, so we could treat it as a self-contained SPA.